docs/modules/operation/pages/alarms/configuring-alarms.adoc

```diff
@@ -83,28 +85,35 @@ However, there are caveats:
 ----
 With this reduction-key, a single alarm would be instantiated for all nodes that were determined by the Poller to be down.
 There would be a single alarm with the count representing the number of nodes down.
-However, the UEI +uei.opennms.org/nodes/nodeUp+ would *not* be a good ''pair wise'' reduction-key for resolving this alarm as it would take only a single ''node up'' to clear all nodes down tracked with this single alarm configuration.
+However, the UEI `uei.opennms.org/nodes/nodeUp` would *not* be a good 'pair wise' reduction-key for resolving this alarm as it would take only a single 'node up' to clear all nodes down tracked with this single alarm configuration.
```

Comment:
I don't think "pairwise" (with or without spaces) appears in product at all.  I think there is just one other reference outside the docs, and that's in a comment in the default opennms.properties file.

```suggestion
However, the UEI `uei.opennms.org/nodes/nodeUp` would *not* be a good 'pairwise' reduction-key for resolving this alarm as it would take only a single 'node up' to clear all nodes down tracked with this single alarm configuration.
```